### ISSUE TITLE
Nagstamon 2.0.1 crashes on app start

### ISSUE THREAD

**Issue by dizzystreak** [HenriWahl/Nagstamon#423]
System: OSX 10.12.6
Nagstamon: 2.0.1

Nagstamon crashes when I try to start it. The Nagstamon app icon appears in my dock, jumps a few times and disappears.
I cannot find any crash reports. The only thing I can find is that ~/.nagstamon/nagstamon.pid is created and left there after the app crashes.
I have cleaned up all app configs and started from scratch, it still crashes.

Can you provide any hints how I could get useful crash data so I can provide more info?

**Comment by HenriWahl** [comment]
Please check latest devel releases from https://nagstamon.ifw-dresden.de/files/incoming/

**Comment by dizzystreak** [comment]
Thank you for the suggestion.
I have discovered where the problem was. It was not Nagstamon itself.
When I tried updating to version 2.1-20170323 a while ago, I deleted everything else but the ~/.nagstamon folder.
SInce then every 2.1 version, including the very latest you proposed, would open with the "Create new server" dialogue. After entering all the information and clicking OK it would forget the entered information and show the dialogue again.
Starting version 2.0.1 would also fail from that point on.
After deleting ~/.nagstamon all versions start up and work fine.

Thank you for the awesome tool.
